packages/web/src/constants/header.constant.ts

```diff
@@ -16,3 +16,26 @@ export const HEADER_NAV = [
     path: "/dashboard",
   },
 ];
+
+export const SIDE_MENU_NAV = [
```

Comment:
I think the Array type is not explicit.
For example, it is hard to predict that SIDE_MENU_NAV[0] is the value for 'Governance'.
How about declaring it as an object?

Reply:
good idea. I used object.